core/iwasm/include/wasm_export.h

```diff
@@ -225,6 +225,26 @@ wasm_runtime_realloc(void *ptr, unsigned int size);
 WASM_RUNTIME_API_EXTERN void
 wasm_runtime_free(void *ptr);
 
+#if WASM_ENABLE_GC != 0
```

Comment:
No need to add WASM_ENABLE_GC in this header file, as developer may include this file and link iwasm with libiwasm.so, and doesn't add macro `WASM_ENABLE_GC` in the make file.